### DISCUSSION TITLE
put a comma between numbers?

### DISCUSSION THREAD

**Discussion by New-three** [jomjol/AI-on-the-edge-device#778] [Q&A]
I only have digital numbers and no analogue ones.
Is it possible to put a comma between the fifth and sixth number?

**Comment by jomjol** [comment]
RTFM :-)

https://github.com/jomjol/AI-on-the-edge-device/wiki/Configuration-Parameter-Details

![grafik](https://user-images.githubusercontent.com/30766535/163662667-17a08321-11c1-470d-9015-abdd2ca17230.png)
